addons/website/models/website_visitor.py

```diff
@@ -28,14 +28,28 @@ class WebsiteTrack(models.Model):
 class WebsiteVisitor(models.Model):
     _name = 'website.visitor'
     _description = 'Website Visitor'
-    _order = 'last_connection_datetime DESC'
+    _order = 'id DESC'
 
-    name = fields.Char('Name')
```

Comment:
I would keep a computed name field, as it leads to cleaned code compared to name_get overrides (we always end up with unwanted code in name_get overrides).